build.sbt

```diff
@@ -217,7 +218,12 @@ lazy val commonSettings =
         "io.netty" % "netty-handler" % nettyV,
         "io.netty" % "netty-codec" % nettyV,
         "io.netty" % "netty-transport-native-epoll" % nettyV,
-      )
+
+        // Jackson is used by openapi and jwks-rsa
```

Comment:
i'm not sure if these libraries should be in common, can we be more precise so that we avoid adding them to unnecessary modules ?

Reply:
But this is the depenedencyOverrides field. We want it here to have the same version in any submodule when imported there.